### PR TITLE
refactor: update Dart package dependencies

### PR DESCRIPTION
**Pull request by kevmoo**
Allow the latest version of analyzer and move from html5lib -> html package

### CODE REVIEW

**Comment by kevmoo** [comment]
@yjbanov @vicb Thoughts?

File: modules/angular2/src/dom/html5lib_adapter.dart [hunk 1]
```diff
@@ -1,8 +1,8 @@
-library angular2.dom.html5adapter;
+library angular2.dom.htmlAdapter;
```
Comment (yjbanov) [inline modules/angular2/src/dom/html5lib_adapter.dart]: `mv html5lib_adapter.dart html5_adapter.dart`

**Comment by yjbanov** [comment]
LGTM, but let's update "html5lib" in file names too.

**Comment by kevmoo** [comment]
Crap! Sorry @mhevery, @yjbanov , @vicb 

I lecture my team about using rebaes/push instead of the button. They make it too easy to press. :sheep:

**Pull request merged**